<?xml version="1.0"?>
<!-- Responsible for launching AK60 motor manager nodes,
will not provide any controllers; just a simple interface-->
<launch>
  <group ns="deimos">
    <param name="robot_description" textfile="$(find deimos_control)/urdf/deimos.urdf" />
    <node pkg="tmotor" type="tmotor_node" name="actuator1" output="screen">
      <param name="id" value="11"/>
      <param name="can_interface" value="vcan0" />
    </node>
    <node pkg="tmotor" type="tmotor_node" name="actuator2" output="screen">
      <param name="id" value="12"/>
      <param name="can_interface" value="vcan0" />
    </node>
    <node pkg="tmotor" type="tmotor_node" name="actuator3" output="screen">
      <param name="id" value="13"/>
      <param name="can_interface" value="vcan0" />
    </node>
    <node pkg="tmotor" type="tmotor_node" name="actuator4" output="screen">
      <param name="id" value="14"/>
      <param name="can_interface" value="vcan0" />
    </node>
  </group>
</launch>